app/code/Magento/Contact/Controller/Index/Post.php

```diff
@@ -101,4 +65,61 @@ private function getDataPersistor()
 
         return $this->dataPersistor;
     }
+
+    /**
+     * @param array $post Post data from contact form
+     * @return void
+     */
+    protected function sendEmail($post)
```

Comment:
That's good to hear. Would you mind if I removed the abstract base controller then and moved the email sending out of the controller? I hesitated because it looked like inheritance was *intended* here and extensions might already use it (although that's improbable)